src/pages/AdminEnrollmentList.js

```diff
@@ -188,19 +227,37 @@ class AdminEnrollmentList extends React.Component {
           </button>
         );
       },
-      Filter: ({ filter, onChange }) => (
-        <select
-          onChange={event => onChange(event.target.value)}
-          value={filter ? filter.value : ''}
-        >
-          <option value="">Tous</option>
-          {toPairs(ADMIN_STATUS_LABELS).map(([key, label]) => (
-            <option key={key} value={key}>
-              {label}
-            </option>
-          ))}
-        </select>
-      ),
+      Filter: ({ filter, onChange }) => {
+        const value = filter
```

Comment:
it looks a lot like the previous ternary, maybe create an helper function you can document that will be easier to understand what it is doing